packages/victory-core/src/victory-util/user-props.js

```diff
@@ -48,6 +48,19 @@ const testIfSafeProp = (key) => {
   return false;
 };
 
+/**
+ * Gets the value from props if a function value is provided
+ * @param {any} value: maybe function value
+ * @param {Object} props: props object
+ * @returns {any} newValue
+ */
+const getValue = (value, props) => {
+  if (typeof value === "function") {
+    return value(props);
+  }
+  return value;
+};
```

Comment:
This is a minor change to the user-provided props that mimics the API of other Victory props and allows us to pass in a function that evaluates the props. 

For example:
```jsx
<VictoryLine aria-label={props => props.label} />
```

I added this small change to this PR because it was useful for testing, but I will make a note to add some additional documentation around this in a separate PR.